### DISCUSSION TITLE
Error Notification

### DISCUSSION THREAD

**Discussion by RandomUser345** [RandomNinjaAtk/docker-lidarr-extended#59] [Ideas]
I'm having the issue of errors, esp. expired Tidal auth, that I'm not aware about and would love to be notified about them.

Thinking about creating a PR with a webhook which gets called on errors and it's up to you if you use it or not.

My first draft would be a simple POST with something like this `{"Event":"Error", "Message":"TidalDownload not working"}`.

Anyone else would find this useful?

**Comment by RandomNinjaAtk** [comment]
That does sound useful, I wish the client had a better auth system. PR welcome, you’d also probably need to include instructions on how to use it.

**Comment by Protektor-Desura** [comment]
I wish there was some kind of notification when your ARL expired so you knew to update it.

**Reply by RandomNinjaAtk** [reply]
The script should just stop working when that happens with an error message. (I think)

**Reply by RandomUser345** [reply]
Yes, and now the scripts also sends to this webhook, if configured. So you just need anything reacting to a http-post and configure the listener as env-variable in docker.
